content/posts/newsletter/2021-04-05.md

```diff
@@ -0,0 +1,179 @@
+
+Arnaud Joset wrote a small tutorial on [how to use Prosody with a HTTP Reverse Proxy](https://blog.agayon.be/prosody_http.html).
+
+Marek Foss from ProcessOne wrote several articles on ejabberd background and setups:
+- [Starting with MQTT protocol and ejabberd MQTT broker](https://www.process-one.net/blog/starting-with-mqtt-protocol-and-ejabberd-mqtt-broker/)
+- [Install and configure MariaDB with ejabberd](https://www.process-one.net/blog/install-and-configure-mariadb-with-ejabberd/)
+- [Getting started with WebSocket API in ejabberd](https://www.process-one.net/blog/getting-started-with-websocket-api-in-ejabberd/)
+- [Publish-Subscribe pattern and PubSub in ejabberd](https://www.process-one.net/blog/publish-subscribe-pattern-and-pubsub-in-ejabberd/)
+
+Alex Akinbia and Ehizojie Ojieb published ["Forensic analysis of open-source XMPP multi-client social networking apps on iOS devices"](https://www.sciencedirect.com/science/article/pii/S2666281721000196?via%3Dihub). They did their study based on Monal 4.5 and Siskin 5.8.1.
+
+## Software news
+
+### Clients and applications
+
+Conversations [2.9.8]((https://github.com/iNPUTmice/Conversations/releases/tag/2.9.8)) and [2.9.9](https://github.com/iNPUTmice/Conversations/releases/tag/2.9.9) have been released. These releases bring better compatibility with non-libwebrtc WebRTC implementations, verification of audio and video (A/V) calls with preexisting OMEMO sessions, and better TOR support.
+
+![Conversations verification A/V](/images/newsletter/march2021/conversations.png "Conversations verification A/V")
+
+[Gajim Development News (March)](https://gajim.org/post/2021-03-28-development-news-march/): Gajim’s new main window is coming together step by step while many core functionalities are being rewritten. This month also brought better accessibility for Gajim’s message input and improved status icon behavior. [Gajim 1.3.1 has been released](https://gajim.org/post/2021-03-01-gajim-1.3.1-released/) as well and they added a setting to explicitly enable GSSAPI authentication and improvement changes.
+
+JSXC - the Javascript XMPP Client got a [funding for group chat calls!](https://twitter.com/jsxc_org/status/1366352995508748294)
+
+[Monal is now on Mastodon](https://fosstodon.org/@Monal)! The [second beta release](https://monal.im/blog/monal-5-beta-2-out/) has been published as well, high-way to Monal 5.0! Besides, a setup has been created to [support the vaccination process](https://vaccine.monal.im/) in many US states, but also Puerto Rico. More than one million notifications have been sent already. XMPP is once again supporting in the pandemic crisis. There were [several blog posts in March](https://monal.im/2021/03/) on this topic, you can start with this selected one](https://monal.im/blog/xmpp-ending-this-pandemic-part-1/). Stay tuned and please support by testing the current beta!
+
+qXMPPconsole - a [browser based XMPP console](https://github.com/voger/qXMPPconsole).
```

Comment:
This line does not really say what happened to the client. Was it released? Did it just pop into existence?